### DISCUSSION TITLE
MuST-C v2 corpus download?

### DISCUSSION THREAD

**Discussion by alumae** [espnet/espnet#5506] [General]
Does anybody know where I could download MuST-C v2? All docs point to https://ict.fbk.eu/must-c/ which is not available any more. Perhaps somebody can share it via Academic Torrents or something like that?

**Comment by brianyan918** [comment]
Looks like it was moved to here: https://mt.fbk.eu/must-c/

I'm going to update the data download scripts now as well.

**Comment by sw005320** [comment]
This is fixed with https://github.com/espnet/espnet/pull/5516
Thanks @brianyan918!
